communications/rfc.rst

```diff
@@ -177,6 +177,15 @@ If an RFC includes API changes, including deprecations, or updates to a data mod
 
 In all of these cases, the RFC is marked as **Flagged** to indicate that it is being considered by the DMCCB.
 
+As a result of the discussion on a **Flagged** RFC, the DMCCB may decide to approve it and move forward with the proposal.
+In this case, the RFC status is set to **Board Recommended**.
+The assignee can then attach the triggering issues and adopt the RFC following the standard process or reopen it changing its status to **Fagged**.
```

Comment:
Fagged -> Flagged

I'm not sure we need to say that last bit though. "following the standard process" is sufficient.

Reply:
So you two are the DMCCB authorities, but:

Should the DMCCB not be considering the RFC *after* the triggered tickets have been created, rather than before?

We've been strict that RFCs may not be adopted before the triggered tickets are created, as there's too much scope for misreading a lengthy discussion or believing consensus has been reached when it hasn't really. Making the triggered tickets is supposed to make the concrete outcome of the RFC clear.

It seems to me that it's that concrete outcome which the DMCCB should be considering.

(And yes, I realise that isn't what was discussed at the meeting earlier; I'm just trying to think through the implications.)

Reply:
That's an interesting idea. Insisting on the triggered tickets being there before CCB weigh in is worth considering. It would mean that CCB would be adopting the RFC directly rather than the reporter.

Reply:
I was just trying to capture the conclusions we reach during the CCB telecons.

@jdswinbank , it seems that you have doubts about the process we are following. Shall I cancel or suspend this PR and bring the topic again for discussion in the next DMCCB?

Reply:
So I have a few different thoughts in response to that —

- As Tim says, I thought it was an interesting idea, and worth suggesting. I didn't intend it to be a demand, or even as strong as a request. However, if you reckon it's worthy of further discussion, I'd be happy to participate in that.
- That said, I will be travelling 2 through 24 March, and won't be at the next DMCCB meeting to discuss it.
- In general (for this and other policy or procedural discussions), my suggestion would be that the best thing to do is for one individual to write out a detailed proposal, thinking through all of the ramifications, before proposing it to the meeting for approval. I'm concerned that *any* proposal we come up with at the tail end of a 30 minute timeboxed DMCCB telecon will be inadequately thought through and leave a bunch of loose ends hanging.

Reply:
I suggest we implement in the dev guide the changes agreed during the DMCCB #3 and #6, and we defer further discussions.

It can be just an AOB point in the next DMCCB, or, if the discussion may be too long, become an RFC.